lib/lz4.h

```diff
@@ -777,6 +778,8 @@ LZ4LIB_API int LZ4_decompress_fast_usingDict (const char* src, char* dst, int or
 LZ4LIB_API void LZ4_resetStream (LZ4_stream_t* streamPtr);
 
 
+/*! Obsolete partial decompress with dict functions */
+LZ4_DEPRECATED("use LZ4_decompress_safe_partial_usingDict() instead") LZ4LIB_API int LZ4_decompress_safe_partial_withPrefix64k(const char* source, char* dest, int compressedSize, int targetOutputSize, int dstCapacity);
```

Comment:
It's too soon to declare the earlier variant "obsolete".
It could be, but that will require more time to declare.

I understand it's "included" in the proposed "usingDict",
but the performance profile of `usingPrefix64k` is much more straightforward to establish than `usingDict`, which will vary more depending on the detected `extDict` vs `prefix` status of the dictionary.

Reply:
Thanks for your review! Yes, should I remove it?

Reply:
Thanks! Done.